components/expenses/ExpenseAttachedFiles.js

```diff
@@ -10,6 +10,14 @@ import UploadedFilePreview from '../UploadedFilePreview';
 
 import ExpenseInvoiceDownloadHelper, { getExpenseInvoiceFilename } from './ExpenseInvoiceDownloadHelper';
 
+const getAttachmentDescription = injectIntl((idx, totalNbFiles, intl) => {
+  if (totalNbFiles === 1) {
+    return intl.formatMessage({ id: 'File.AttachedFile', defaultMessage: 'Attached file' });
+  } else {
+    return intl.formatMessage({ defaultMessage: 'Attached file {number}' }, { number: idx + 1 });
+  }
+});
+
```

Comment:
I see your original point now! In the latest commit I've reverted the changes to `UploadedFilePreview` and implemented your last suggestion to be consistent with what's done elsewhere in the codebase. Unfortunately I still can't see my expense on `localhost:3000` and I'm hesitant to create one in case it gets sent to the (real) backend. But these changes pass command-line build and test stages.